build.sbt

```diff
@@ -7,7 +7,7 @@ scalaVersion in ThisBuild := "2.12.2-ebe1180-SNAPSHOT" // from https://github.co
 scalaBinaryVersion in ThisBuild := "2.12"
 
 scalacOptions in ThisBuild ++=
-  Seq("-deprecation", "-unchecked", "-Yno-imports", "-language:higherKinds")
+  Seq("-deprecation", "-unchecked", "-language:higherKinds")
```

Comment:
Again, an artifact of this being ported from an external repo: https://github.com/pathikrit/scalgos/commit/bd1d794ffb573a563f155ff845bdedcf453542bd

Will put back all imports